tools/package_gcsfuse_docker/Dockerfile

```diff
@@ -37,7 +37,12 @@ ARG DEBFULLNAME="GCSFuse Team"
 # Build Arg for building through a particular branch/commit. By default, it uses
 # the tag corresponding to passed GCSFUSE VERSION
 ARG BRANCH_NAME="v${GCSFUSE_VERSION}"
-RUN git checkout "${BRANCH_NAME}"
+ARG GIT_BUILD_TAG
```

Comment:
NIT: Another possible way to do this would have been not making this change and just passing `--build-arg BRANCH_NAME ="v$RELEASE_VERSION_TAG"` while building the dockerfile. We probably could have renamed `BRANCH_NAME` to something more generic like `GIT_CHECKOUT`

Reply:
+1, we should use BRANCH_NAME and rename it later as separate task at all places

Reply:
Ohh, got it.  "v${GCSFUSE_VERSION}" is the default value. Thanks!